### PR TITLE
Use npm instead of Yarn

### PR DESCRIPTION
**Pull request by nickmccurdy**

### CODE REVIEW

**Comment by MatanBobi** [comment]
I'm actually a big fan of `yarn` so I'm trying to understand why we want to switch it here :)

**Comment by nickmccurdy** [comment]
I prefer Yarn too, but the organization prefers npm and I think it's better to keep things consistent so contributors to libraries only need to learn npm to contribute to the docs.

**Comment by MatanBobi** [comment]
> I prefer Yarn too, but the organization prefers npm and I think it's better to keep things consistent so contributors to libraries only need to learn npm to contribute to the docs.

Got it, didn't know that the organization prefers npm :)
Thanks for clarifying this!

File: .gitignore [hunk 1]
```diff
@@ -7,8 +7,6 @@ lib/core/MetadataBlog.js
 
 website/translated_docs
 website/build/
-website/yarn.lock
```
Comment (MatanBobi) [inline .gitignore:10]: Do we want to add `package-lock` to gitignore in order to keep the "no lock file approach" or are we planning to fix it with this PR?

**Review by MatanBobi — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (alexkrolick) [reply .gitignore:10]: I think we want to allow the lockfile for the site to alleviate build indeterminacy.

File: .gitignore [hunk 2]
```diff
@@ -7,8 +7,6 @@ lib/core/MetadataBlog.js
...
-website/node_modules
```
Comment (alexkrolick) [inline .gitignore:11]: I am guessing the `node_modules` above catches this, making it redundant?

Comment (nickmccurdy) [reply .gitignore:11]: Yes

**Pull request merged**